MyPiggyBank.Test/Account/AccountProfileTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoMapper;
+using MyPiggyBank.Core.Communication.Account.Mappings;
+using Xunit;
+
+namespace MyPiggyBank.Test.Account
+{
+    public class AccountProfileTest
+    {
+        private readonly MapperConfiguration _mapperConfiguration;
+
+        public AccountProfileTest()
+        {
+            _mapperConfiguration = new MapperConfiguration(c =>
+            {
+                c.AddProfile<AccountProfile>();
+            });
+        }
+
+        [Fact]
+        public void Profile_ShouldBeValid()
```

Comment:
Skąd taka śmieszna nazwa metody?

Reply:
To jest nazwa metody dla testów, która powinna się określać co testuje i jaki powinien być rezultat. Sprawdza poprawność profilu mapującego, ponieważ mogą wystąpić propercje niezmapowane i zarazem niezignorowane. W takiej sytuacji mapper się po prostu wywróci. Dlatego profil może być nie prawidłowy, bo będzie wykrzaczał mappera a my oczekujemy jego poprawnego działania. Jesli jest jakaś dobra sugestia na lepszą nazwę to spoko.